packages/bp/src/core/messaging/messaging-service.ts

```diff
@@ -118,6 +118,10 @@ export class MessagingService {
   }
 
   async receive(event: MessageNewEventData) {
+    if(!this.channelNames.includes(event.channel)) {
+      return
```

Comment:
So we silently fail without warning? Fun debugging time

Reply:
This is fine. It's a fix for not receiving web messages twice. It's already implemented in another PR so it shouldn't be here though.

Reply:
@allardy It only means that if you receive messages that are related to a channel that isn't supported by Botpress or Messaging, it will be discarded. I don't this it is necessary to add logging here as you should not receive messages from unknown channels!